packages/frontend/src/components/nft/NFTDetail.js

```diff
@@ -0,0 +1,196 @@
+import React, { useEffect, useState } from 'react';
```

Comment:
It would be really sweet if you wanted to create a `NFTDetailWrapper` to separate async behavior. Basically keep the actions and selectors in the wrapper. You can see how it was done recently here for the app login flow with `ConfirmLoginWrapper` and `ConfirmLogin` component as a child: https://github.com/near/near-wallet/tree/master/packages/frontend/src/components/login/v2

Reply:
will do